packages/dara-components/changelog.md

```diff
@@ -6,6 +6,7 @@ title: Changelog
 
 -   `Chat` component now shows the user who has written a message
 -   `Chat` component now displays its messages content as `Markdown`.
+-   `Chat` sends a notification when a new message is addded to it.
```

Comment:
That's not entirely true since it requires configuring, should we expand about it here a bit? Maybe a small snippet on how a user can configure what happens to new messages and that it can be used to e.g. send notifications

Reply:
Forgot to update changelog after changes, good catch!